Banking/src/main/java/com/example/Banking/service/AccountDaoImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.example.Banking.service;
+import com.example.Banking.exception.IdNotFoundException;
+import com.example.Banking.model.Account;
+import com.example.Banking.repository.AccountRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class AccountDaoImpl implements AccountDao {
+
+    @Autowired
+    private AccountRepository accountRepository;
+
+    private static final String  ERROR_MESSAGE = "Account is not Found for this ";
+
+    @Override
+    public List<Account> getAll() {
+        return accountRepository.findAll();
+    }
+
+
+    public Account getAccount(long acNumber) throws IdNotFoundException {
+
+            return accountRepository.findById(acNumber).orElseThrow(() -> new IdNotFoundException(ERROR_MESSAGE + " "+ acNumber));
+
+    }
+
+    @Override
+    public void insertAccount(Account account) {
+        accountRepository.save(account);
+
+    }
+
+
+    @Override
+    public void deleteAccount(long acNumber) throws IdNotFoundException {
+        if(accountRepository.existsById(acNumber)) {
+            accountRepository.deleteById(acNumber);
+        }
+        else
+        {
+            throw new IdNotFoundException(ERROR_MESSAGE+ " "+ acNumber);
+        }
+    }
+
+    @Override
+    public void updateAccount(Account account, long acNumber) throws IdNotFoundException {
+        Account accounts = accountRepository.findById(acNumber).orElseThrow(() -> new IdNotFoundException(ERROR_MESSAGE +" "+acNumber));
+        accounts.setAccountNumber(account.getAccountNumber());
```

Comment:
There are some things which shouldn't be modified by update operation. Like accountNumber which is the primary key

Reply:
> I have some comments on renaming the stuff. You can merge this code after renaming it.
I have renamed the stuff. I will merge.